### PR TITLE
10 40K

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/acolyte_hybrid.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Acolyte Hybrid
+ManaCost:2 R
+Types:Creature Tyranid Human
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigDestroy | TriggerDescription$ Heavy Rock Cutter — Whenever CARDNAME attacks, destroy up to one target artifact. If an artifact is destroyed this way, its controller draws a card.
+SVar:TrigDestroy:DB$ Destroy | ValidTgts$ Artifact | TargetMin$ 0 | TargetMax$ 1 | RememberDestroyed$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | Defined$ RememberedController
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/acolyte_hybrid.txt:7]: Needs DBCleanup

File: forge-gui/res/cardsfolder/upcoming/marneus_calgar.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Marneus Calgar
+ManaCost:2 W U B
+Types:Legendary Creature Astartes Warrior
+PT:3/5
+K:Double Strike
+T:Mode$ TokenCreated | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ DBDraw | TriggerDescription$ Master Tactician — Whenever one or more tokens enter the battlefield under your control, draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/marneus_calgar.txt:6]: Wrong trigger, see _Woodland Champion_

File: forge-gui/res/cardsfolder/upcoming/the_first_tyrannic_war.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:The First Tyrannic War
+ManaCost:2 G U R
+Types:Enchantment Saga
+K:Saga:3:DBChangeZone,DBDouble,DBDouble
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | SubAbility$ DBCounter | ChangeType$ Creature.YouCtrl | Optional$ True | RememberChanged$ True | SpellDescription$ You may put a creature card from your hand onto the battlefield. If its mana cost contains {X}, it enters the battlefield with a number of +1/+1 counters on it equal to the number of lands you control.
+SVar:DBCounter:DB$ PutCounter | Defined$ Remembered | IsPresent$ Creature.IsRemembered+hasXCost | CounterType$ P1P1 | CounterNum$ X |
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_first_tyrannic_war.txt:6]: It should ETB with the counters, might need more support

**Comment by Simisays** [comment]
@tool4ever You're right in that ETB$ wasn't needed. However without that the card seems to be working fine. Here's a pic
image

File: forge-gui/res/cardsfolder/upcoming/the_first_tyrannic_war.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/the_first_tyrannic_war.txt]: Why resolve?
It matters for interaction like "Whenever a creature with power X or greater enters the battlefield"

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/the_first_tyrannic_war.txt]: Oh I understand now, I'll just create a new PR with it later.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**